docs/refguide/api.md

```diff
@@ -520,6 +520,22 @@ Use this if you want to check whether you are using computed properties without
 configure({ computedRequiresReaction: true });
 ```
 
+#### `observablesRequiresReaction: boolean`
+Warns about any unobserved observable access.  
+Use this if you want to check whether you are using observables without a reactive context (eg react component without observer wrapping).
+
+```javascript
+configure({ observablesRequiresReaction: true });
+```
+
+#### `derivationRequiresObservable: boolean`
+Warns when a derivation (reaction, autorun, observer react component) is created without any observable access.  
+Use this to check whether you are unneededly wrapping react components with `observer`, or to find possible related bugs.
```

Comment:
Not sure if to mention React directly like that in here, it might sound like it's only use case :)

Besides, this is applied globally, is it influence everything, even manually created derivations inside the components.

Reply:
It's the most down to earth example i could come up with.
i can write a whole blog post on these 2 options, but i'm not sure if that api reference is the place. i can omit that it its a problem